test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/IntegrationTests.cs

```diff
@@ -49,22 +65,88 @@ public void ExportResultIsSuccess(OtlpExportProtocol protocol, string endpoint)
                 Protocol = protocol,
             };
 
-            var otlpExporter = new OtlpTraceExporter(exporterOptions);
-            var delegatingExporter = new DelegatingTestExporter<Activity>(otlpExporter);
-            var exportActivityProcessor = new SimpleActivityExportProcessor(delegatingExporter);
+            DelegatingTestExporter<Activity> delegatingExporter = null;
 
             var activitySourceName = "otlp.collector.test";
 
             var builder = Sdk.CreateTracerProviderBuilder()
-                .AddSource(activitySourceName)
-                .AddProcessor(exportActivityProcessor);
+                .AddSource(activitySourceName);
+
+            OtlpTraceExporterHelperExtensions.AddOtlpExporter(
```

Comment:
I am unsure what this change accomplishes... Could you elaborate what was broken, and how this would help catching it?

Reply:
So the actual bug in RC3 was with `MetricReaderOptions`. The default got set to manual so `OtlpMetricExporter` stopped doing its work. The tests the way they were wouldn't have actually caught that bug because they were manually managing the processor/reader setup. In order to make them more true integration tests I wanted to use more of the actual code users will be executing which is the builder extensions. What I did here is make the extensions internal so I could call them and then added a mechanism so I could wrap the actual exporter with `DelegatingTestExporter` in order to validate the results in the tests.

Reply:
We added "Alan dont screw up OTLP exporter" Tests 🤣 https://github.com/open-telemetry/opentelemetry-dotnet/pull/2982/commits/1a53185ae8cf4f188eca5983c3f5531bba0580e6
Thought that was enough!